rubrics/tool/src/main/frontend/webcomponents/sakai-rubric-association.js

```diff
@@ -10,7 +10,8 @@ class SakaiRubricAssociation extends SakaiElement {
     super();
 
     this.configurationOptions = [];
-    SakaiRubricsLanguage.loadTranslations();
+
+    SakaiRubricsLanguage.loadTranslations().then(result => this.i18nLoaded = result );
```

Comment:
So this definitely not going to work in IE, right?

Reply:
We need to use webpack to build a bundle if we want it to work in IE. It's the arrow functions, mainly, as we are using a polyfill which patches in promises and fetch, stuff like that.